bundles/EcommerceFrameworkBundle/IndexService/ProductList/ElasticSearch/AbstractElasticSearch.php

```diff
@@ -1135,6 +1159,65 @@ protected function doLoadGroupByValues()
         $this->preparedGroupByValuesLoaded = true;
     }
 
+    /**
+     * Deep search for buckets in result aggregations array, as the structure of the result array
+     * may differ dependent on the used aggregations (i.e. date filters, nested aggr, ...)
+     *
+     * @param array $aggregations
+     * @return array
+     */
+    private function searchForBuckets(array $aggregations)
```

Comment:
shouldn't we make these methods protected to make overwriting possible?

Reply:
makes sense - resolved